.prettierrc.yaml

```diff
@@ -0,0 +1,13 @@
+printWidth: 80
+semi: false
+trailingComma: all
+overrides:
+  - files:
+      - README.md
+    options:
+      proseWrap: preserve
```

Comment:
Review: Preserve is the default so this is no-op, mainly included it because it existed in the previous config as always, but applying always now would produce a significant diff. We can revisit what we want to do with prose at a later time. It has pros and cons (reflowing text causes larger diffs when editing).